libbeat/docs/generalconfig.asciidoc

```diff
@@ -103,3 +103,70 @@ processors in your config.
 
 Sets the maximum number of CPUs that can be executing simultaneously. The
 default is the number of logical CPUs available in the system.
+
+[float]
+[[configuration-internal-queue]]
+=== Configure the internal queue
+
+The `queue` namespace configures the queue type to be used.
+
+The Elastic Beats employ an internal queue for events to be published. The
```

Comment:
Crap weasel. Looks like some of my comments didn't get saved when I submitted the review. I'd change this to say:

```
{beatname_uc} uses an internal queue to store events before publishing them. 
```